CHANGES.rst

```diff
@@ -172,6 +172,12 @@ Bug Fixes
 
 - ``astropy.utils``
 
+  - Fix to allow the C-based _fast_iterparse() VOTable XML parser to
+    relloc() its buffers instead of overflowing them.  [#5824] [#5869]
```

Comment:
PR  number needs to be at the end of the entry. Change to `[#5824, #5869]` and move it to after the parenthesis. (Not sure how strict is the bot but consistent format does not hurt.)